internal/kafka/internal/api/admin/private/api/openapi.yaml

```diff
@@ -373,16 +373,24 @@ components:
     Error:
       properties:
         reason:
+          description: Human-readable description of the error. Intended for human
```

Comment:
The intention of this description is that `reason` is a human-readable text. It cannot be used for programmatic error handling.
This means it offers no guarantees and it is subject to change.

Reply:
Yes. See https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1408#discussion_r1029323363.

The important part about the description of the reason field is that it is intended for human consumption. So we are free to change the values we return in the responses at any time. It is not part of the API contract